app/models/clinic.rb

```diff
@@ -17,13 +17,33 @@ class Clinic
   field :active, type: Boolean, default: true
   field :accepts_naf, type: Boolean, default: false
   field :gestational_limit, type: Integer
-  field :costs_9wks, type: Integer
+  field :costs_05wks, type: Integer
```

Comment:
Let's go without leading zeroes, if that's okay with you. Happy to handle since this is my own pedantic junk, if you prefer.